src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +502,56 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if is_fusion_transcript_segment:
+            # Check if is_fusion_transcript_segment is given
+            alt_ac = self.sr.chromosome_to_acs(chromosome)[0][0]
+            if alt_ac is None:
+                return self._return_warnings(
+                    resp, "Chromosome could not be converted to an accession"
+                )
+            mane_transcripts = self.mane_transcript_mappings.get_gene_mane_data(gene)
+            if mane_transcripts:
+                transcript = mane_transcripts[0]["RefSeq_nuc"]
+
+            else:
+                result = await self.uta_db.get_transcripts(gene=gene, alt_ac=alt_ac)
+                if len(result) > 0:
+                    transcript = result[0]["tx_ac"][0]
+                else:
+                    # Run if gene is for a noncoding transcript
+                    query = f"""
+                        SELECT DISTINCT tx_ac
+                        FROM {self.uta_db.schema}.tx_exon_aln_v
+                        WHERE hgnc = '{gene}'
+                        AND alt_ac = '{alt_ac}'
+                        """  # noqa: S608
+                    result = await self.uta_db.execute_query(query)
+                    transcript = result[0]["tx_ac"]
+
+            tx_genomic_coords = await self.uta_db.get_tx_exons_genomic_coords(
```

Comment:
Need to check if value exists first

Reply:
I'm not seeing where you're checking to see if `tx_genomic_coords` exists

Reply:
Once you do, just create a new variable or set to the first object since you only reference the first object